BUILD

```diff
@@ -68,7 +73,10 @@ cc_library(
         "upb/encode.h",
         "upb/upb.h",
     ],
-    copts = COPTS,
```

Comment:
Can we put the select inside the `COPTS` var so we don't need to repeat it for every cc_library()?

Reply:
I don't think we can, no. I can try.

Reply:
That's a no by the way. Looks like select can only work either as an actual property, or as the return value of a skylark macro. We *could* push it into a skylark macro, something along these lines:

```python
def get_copts():
  return select({
    "windows": [],
    "//condition:default": ["-Werror"],
  })

Reply:
I'm going to go ahead and merge as is, since there's no direct way to do what you were suggesting. There's a reason why we wrap all of our libraries and binaries in macros on grpc :)